### DISCUSSION TITLE
ASK FOR HELP FOR HOW TO USE

### DISCUSSION THREAD

**Discussion by rzp-phd** [NSRingo/.github#14] [Q&A 问答] [answered]
I used the Little Rocket one-click import, and the module showed that it was added successfully, but the function still cannot be used
![54ee8301282c9410080ec8732170e75](https://github.com/user-attachments/assets/b011f761-33c4-4181-b5ff-d5051f171e1b)

**Comment by VirgilClyne** [comment, marked as answer]
Refer to the basic usage tutorial of Shadowrocket:
https://t.me/ShadowrocketApp
